spec/unit/peep_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'peep'
+
+describe Peep do
+
+  describe '.create' do
+
+    it 'allows you to create a message' do
+      expect{ Peep.create(message: 'How is everyone doing?', posted: DateTime.now.strftime('%d/%m/%_Y %_H:%_M')) }.to change{ Peep.all.length }.by (1)
+    end
+
+  end
+
+  describe '.all' do
+
```

Comment:
can you test that `.all `stores more than one message, i.e. set up the test with multiple `Peep.create` statements?